File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.cpp
class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        int count=0;
        for(int i=0; i<words.size(); i++)
        {
            string s1=words[i].substr(0,pref.size());
            {
                if(s1==pref)
                {
                    count++;
                }
            }
        }
        return count;
    }
};

File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.java
class Solution {
    public int prefixCount(String[] words, String pref) {
     int count=0;
        for(int i=0; i<words.length;i++)
        {
            String str=words[i];
            if(str.length()<pref.length())
            {
                continue;
            }
            if(str.startsWith(pref))
            {
                count++;
            }
        }
        return count;
    }
}